test/minimal_init.lua

```diff
@@ -0,0 +1,73 @@
+local execute = vim.api.nvim_command
+local fn = vim.fn
+
+local install_path = "/tmp/site/pack/packer/opt/packer.nvim"
+
+if fn.empty(fn.glob(install_path)) > 0 then
+  execute("!git clone https://github.com/wbthomason/packer.nvim " .. install_path)
+  execute "packadd packer.nvim"
+end
+
+execute "packadd packer.nvim"
+
+vim.o.termguicolors = true
+
+-- Only required if you have packer in your `opt` pack
```

Comment:
do we need this comment? I think we don't need it because the auto install thingy above put it on `opt` already

Reply:
Nope! Just copied straight from packer.